index.html

```diff
@@ -283,20 +283,244 @@ <h1 class="contact-me">Contact me</h1>
             <textarea name="message" id="message" placeholder="Write your message here..." maxlength="500"
                 required></textarea>
 
-            <button type="submit" class="button">Get in touch</button>
+            <button type="submit" class="submit-button">Get in touch</button>
+            <div id="error-message"></div>
         </form>
     </section>
+    <!-- mobile menu -->
     <div class="mobile-menu hidden">
         <div class="x-btn">
             <a id="closeMenuBtn" href="#" onclick="return false;"><img src="./icons/Enabled.png" alt=""></a>
         </div>
-        
+
         <div class="menu-lists">
             <a id="menu1" href="#portfolioSection">Portfolio</a>
             <a id="menu2" href="#aboutSection">About</a>
             <a id="menu3" href="#contactSection">Contact</a>
         </div>
     </div>
+    <!-- popup window -->
+    <div class="popupWindow1 hidden">
+        <div class="popupWindowContainer">
+            <div class="popupTop">
```

Comment:
Hello @Frederick-Ye,
You did a awesome job replicating the figma design. 💯 👍 However, according to the project requirement, you need to use a JavaScript array to store all of the information for all projects and render dynamically instead of hard coding it. Kindly refactor the project sections for dynamic rendering.